model/import/CsvItemImporter.php

```diff
@@ -65,7 +71,9 @@ public function getLabel()
      */
     public function getForm()
     {
-        $form = new CsvImportForm();
+        $form = new CsvImportForm([], [
+            'classUri' => $this->request->getParsedBody()['classUri']
+        ]);
```

Comment:
If we do not indent, the first impression is that `CsvImportForm` receives a single parameter.

```suggestion
        $form = new CsvImportForm(
            [], 
            [
                'classUri' => $this->request->getParsedBody()['classUri']
            ]
        );
```